stac-gen/handler.py

```diff
@@ -0,0 +1,75 @@
+from cmr import GranuleQuery
+import json
+import pystac
+from pystac.utils import str_to_datetime
+from shapely.geometry import shape
+
+
+def try_int(x):
+    try:
+        return int(x)
+    except ValueError:
+        return x
+
+
+def create_item(entry):
+    bbox = [try_int(x) for x in entry["boxes"][0].split(" ")]
```

Comment:
[rio-stac ](http://devseed.com/rio-stac/) has some really handy features that parse a lot of this stac metadata right out of the tif so I think it's worth considering in the pipeline. If we aren't executing this in a lambda that already has the cog open, we could still use it with vsis3.

Of course we're still going to want to add whatever metadata is relevant for the item and relating it back to cmr somehow.